package.json

```diff
@@ -11,6 +11,7 @@
         "README.md"
     ],
     "scripts": {
+        "start": "npm run styleguide",
```

Comment:
Added this by convention as I expected to be able to run `npm start`, aliases the existing styleguide command.